lib/main.dart

```diff
@@ -1,125 +1,218 @@
+          SingleChildScrollView(
+            // this permit can be scroll
+            scrollDirection: Axis.horizontal, // in the horizontal axis
+            padding: const EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 0.0),
+            child: Row(
+              children: [
+                const Text('Genres',
+                    style: TextStyle(
+                      color: Colors.white,
+                      fontSize: 16,
+                    )),
+                const SizedBox(
+                  width: 10,
+                ),
+                Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: movie.genres
+                        .map((genre) => Padding(
+                              padding: const EdgeInsets.all(4.0),
+                              child: FilterChip(
+                                  // i used this to show the genres, normally is used to checkbox
+                                  shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(40),
+                                    side: const BorderSide(
+                                        width: 2, color: Colors.indigo),
+                                  ),
+                                  backgroundColor: Colors.black87,
+                                  label: Text(
+                                    genre,
+                                    style: const TextStyle(
+                                      color: Colors.white54,
+                                      fontSize: 15,
+                                    ),
+                                  ),
+                                  onSelected: (b) {}),
+                            ))
+                        .toList()),
+              ],
+            ),
+          ),
+          const Padding(
+            padding: EdgeInsets.all(10.0),
+            child: Text(
+              'Description',
+              style: TextStyle(
+                  color: Colors.white,
+                  fontWeight: FontWeight.w600,
+                  fontSize: 16.0),
             ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headlineMedium,
+          ),
+          Padding(
+            padding: const EdgeInsets.all(10.0),
+            child: Text(
+              movie.overview,
+              style: const TextStyle(
+                  color: Colors.white54,
+                  fontWeight: FontWeight.w600,
+                  fontSize: 14.5),
             ),
-          ],
-        ),
+          ),
+          LikeButton(movie.countLikes),
+        ],
       ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
+    ));
+  }
+}
+
+class LikeButton extends StatefulWidget {
+  int like;
```

Comment:
I'd move this to the `State`, as that class is the one responsible of holding that value